twingraph/src/main/openapi/twingraph.yaml

```diff
@@ -86,6 +86,28 @@ paths:
             application/json:
               schema:
                 type: string
+  /organizations/{organization_id}/twingraph/listgraphs:
```

Comment:
In order to be consistent, please rename the endpoint to `/organizations/{organization_id}/twingraphs`